src/test/scala/org/broadinstitute/dsde/agora/server/dataaccess/authorization/TestAuthorizationProvider.scala

```diff
@@ -12,8 +13,8 @@ import org.broadinstitute.dsde.agora.server.model.AgoraEntity
 object TestAuthorizationProvider extends AuthorizationProvider with AgoraTestData{
 
   // local permission storage.
-  val entityPermissions = new HashMap[String, AgoraPermissions]()
-  val namespacePermissions = new HashMap[String, AgoraPermissions]()
+  val entityPermissions = new mutable.HashMap[String, AgoraPermissions]()
```

Comment:
It always was. This is just a scala convention for calling out mutable collections in the instantiation. (A mild form of shaming, but mostly just to make it obvious you are dealing with mutable collection without looking at the imports)